integrations/gitlab-v2/integration.py

```diff
@@ -23,12 +32,54 @@ class ProjectResourceConfig(ResourceConfig):
     selector: ProjectSelector
 
 
-class GitlabPortAppConfig(PortAppConfig):
-    resources: list[ProjectResourceConfig | ResourceConfig] = Field(
-        default_factory=list
+class FilesSelector(BaseModel):
+    path: str = Field(
+        default="",
+        alias="path",
+        description="Specify the path to match files from",
     )
```

Comment:
I think there shouldn't be an option to not define a path. lets make it required